charts/refarch-templates/ci/test-values.yaml

```diff
@@ -0,0 +1,30 @@
+modules:
```

Comment:
was ist das für ein Ordner? den gibt es hier nicht https://helm.sh/docs/topics/charts/#the-chart-file-structure

Vielleicht die Testdatei zum Testen in ein seperates Testchart aufteilen

Reply:
würde es auch so lassen wie es ist.